src/api/app/controllers/statistics_controller.rb

```diff
@@ -207,8 +207,9 @@ def active_request_creators
     @stats = []
     reqs.sort.each do |month, requests|
       monstats = []
-      requests.group_by(&:creator).sort.each do |creator, list|
-        monstats << [creator, User.find_by_login(creator).email, list.length]
+      requests.group_by(&:user_id).sort.each do |user_id, list|
+        user = User.find_by_login(user_id)
```

Comment:
This will make one sql request per each user but should only make one sql request per all users i.e.
```ruby
user_ids = requests.group_by(&:user_id)
monstats = 
  User.where(id: user_ids).map do |user|
    [user.login, user.email, list.length]
  end
```